lib/index.js

```diff
@@ -974,3 +984,23 @@ exports.shallow = function (source) {
 
     return target;
 };
+
+
+exports.promiseWrap = function (bind, method, args, transform) {
+
+    return new Promise((resolve, reject) => {
+
+        const callback = function (error, result) {
+
+            if (Array.isArray(transform)) {
+                result = exports.transform(Array.from(arguments), transform);
+                error instanceof Error ? reject(result) : resolve(result);
```

Comment:
it's good to be there for such use cases https://github.com/hapijs/catbox/blob/master/lib/policy.js#L217 when sending an error but also some other data